demo/b2share_demo/cli.py

```diff
@@ -93,3 +93,43 @@ def load_config(verbose, force):
     if verbose > 0:
         click.secho('Configuration file "{}" created.'.format(
             instance_config_path), fg='green')
+            
+@demo.command()
+@with_appcontext
+@click.option('-v', '--verbose', count=True)
+@click.option('-d','--download', is_flag=True, default=False)
+@click.option('-l','--limit', default=None)
+@click.argument('token')
+@click.argument('download_directory')
+def import_v1_data(verbose, download, token,
+         download_directory,limit):
+    if verbose:
+        click.secho("Importing data from b2share.eudat.eu to this instance")
+        os.chdir(download_directory)
+        if download:
+            if verbose:
+                click.secho("Cleaning download directory %s" % download_directory)
```

Comment:
Removing files like this is potentially dangerous. Isn't it better to just quit if the directory is not empty?